app/models/custom_button.rb

```diff
@@ -133,7 +141,11 @@ def self.parse_uri(uri)
   end
 
   def self.button_classes
-    return BUTTON_CLASSES
+    @button_class_names ||= BUTTON_CLASSES.collect(&:name)
```

Comment:
@matthewd is the suggestion:

```ruby
def self.button_classes
  BUTTON_CLASSES.collect(&:name)
end
```

or

```ruby
BUTTON_CLASSES = [...].freeze
BUTTON_CLASS_NAMES = BUTTON_CLASSES.collect(&:name).freeze

def self.button_classes
  BUTTON_CLASS_NAMES
end
```

although, I do like the idea of freezing these. (though I am not sure if freeze matters for anything other than strings.